File: trunk/ShadowFrameworkLite20_OpenGL20/src/shadow/pipeline/openGL20/SFGL20Pipeline.java
package shadow.pipeline.openGL20;

import shadow.pipeline.SFPipeline;
import shadow.pipeline.expression.SFExpressionGeneratorKeeper;

public class SFGL20Pipeline {
	public static void setup(){
		SFPipeline.setSfProgramBuilder(new SFGL20ProgramBuilder());
		SFPipeline.setSfPipelineGraphics(new SFGL20PipelineGraphics());
		SFPipeline.setSfPipelineMemory(new SFGL20PipelineMemory());
		SFPipeline.setSfTexturePipeline(new SFGL20TexturePipeline());
		SFExpressionGeneratorKeeper.getKeeper().setGenerator(new SFGL20ExpressionGenerator());
	}
}

File: trunk/ShadowFrameworkJS/SF20OpenGL20Graphics/src/shadow/pipeline/openGL20/SFGL20Pipeline.js


SFGL20Pipeline.prototype = {

	setup:function(){
	SFPipeline.setSfProgramBuilder(new SFGL20ProgramBuilder());//Warning: Not well Identified 
	SFPipeline.setSfPipelineGraphics(new SFGL20PipelineGraphics());//Warning: Not well Identified 
	SFPipeline.setSfPipelineMemory(new SFGL20PipelineMemory());//Warning: Not well Identified 
	SFPipeline.setSfTexturePipeline(new SFGL20TexturePipeline());//Warning: Not well Identified 
	SFExpressionParser.getParser().setGenerator(new SFGL20ExpressionGenerator());//Warning: Not well Identified 
	}

};